values.yaml

```diff
@@ -63,6 +63,9 @@ global:
   # Used to enable Celery autoscaling
   kedaEnabled: false
 
+  # make fluentd to gather logs from all avaialble namespaces
+  preCreatedNamespacesEnabled: false
```

Comment:
I am generally worried about using slightly different flag names for the same feature in different places. Eg: we are calling it manualNamespaceNames in Houston configs, and calling it preCreatedNamespacesEnabled here and possibly something else in Commander. Can we standardize on a name across all our configs?

Reply:
Also, this is a global flag which is currently used in FluentD to gather logs from all namespaces, but it could be used in other places as well. So don't just restrict your comment for that.

Reply:
sounds good.